### PR TITLE
Feat: Proposal Search

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

**Comment by jeysal** [comment]
can't comment on the backend code but seems to work

**Comment by jeysal** [comment]
https://487.hub.asap.science/news-and-events is somehow broken though while it works on dev and prod? o.O

**Comment by kmaid** [comment]
Looks like an issue with cors?

**Comment by fampinheiro** [comment]
I had a look onto our deployment, and this endpoint was never invoked. AWS is playing tricks on us.

**Review by fampinheiro — APPROVED** [review]

File: apps/asap-server/src/controllers/research-outputs.ts [hunk 1]
```diff
@@ -59,26 +59,35 @@ export default class ResearchOutputs {
...
   }): Promise<ListResearchOutputResponse> {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const { search, filter, ...opts } = options;
+    const { search, filter, take = 8, skip = 0 } = options;
+
+    const searchQ = (search || '')
+      .split(' ')
+      .filter(Boolean)
+      .reduce((acc: string[], word: string) => {
+        return acc.concat(`contains(data/firstName/iv, '${word}')`);
```
Comment (kmaid) [inline apps/asap-server/src/controllers/research-outputs.ts:70]: Should first name be here?

**Pull request merged**